package.json

```diff
@@ -86,6 +86,7 @@
     "@babel/preset-flow": "7.8.3",
     "@babel/preset-react": "7.0.0",
     "@babel/register": "7.0.0",
+    "@dump247/storybook-state": "^1.6.1",
```

Comment:
@aleksandardjordjeviciohk please use fixed version here -> `^1.6.1` should be changed to `1.6.1`

Reply:
Updated